refl1d/experiment.py

```diff
@@ -110,12 +110,15 @@ def update(self):
 
     def residuals(self):
         if 'residuals' not in self._cache:
+            # Trigger reflectivity calculation even if there is no data to
+            # compare against so that we can profile simulation code, and
+            # so that simulation smoke tests are run more thoroughly.
+            QR = self.reflectivity()
```

Comment:
This seems like an important change, outside the refactor with numba... I hope this is in the other branches too

Reply:
Not super important. In the past I've forced the call directly within the model file, but this didn't work when profiling.